src/plugins/teleport-plugin-react-inline-styles/index.ts

```diff
@@ -40,9 +40,11 @@ export const createPlugin: ComponentPluginFactory<InlineStyleConfig> = (config)
         // Nested styles are ignored
         const rootStyles = cleanupNestedStyles(style)
         const inlineStyles = transformDynamicStyles(rootStyles, (styleValue) => {
-          return new ParsedASTNode(
-            t.memberExpression(t.identifier('props'), t.identifier(styleValue.content.id))
-          )
+          const expression =
+            styleValue.content.referenceType === 'state'
```

Comment:
same as above. I think we can extract a helper function

Reply:
Yes I agree, we are doing the same kind of check in a couple of style plugins 👍